spector/frontend/src/routes/sports.js

```diff
@@ -14,19 +15,39 @@ const Sports = () => {
   }
 
   return (
-    <main>
-      <h2>Welcome to the {sportName} page</h2>
-      <ul>
-        {data.map((item, index) => (
-          <ul key={index}>
-            <li>{item.name}</li>
-            <li>{item.description}</li>
-            <li>{item.sport}</li>
-            <li>{item.id}</li>
+    <div className="SportPageItems">
+      <h2 className="socSportPageTitle">Welcome to the {sportName} page</h2>
+      
+      <ul className="socPageLayout">
+
+        <li>
+          <ul className="socIndividualBox">
+            {data.map((item, index) => (
+              <ul key={index}>
+                <li>{item.name}</li>
+                <li>{item.description}</li>
+                <li>{item.sport}</li>
+                <li>{item.id}</li>
+              </ul>
+            ))}
           </ul>
-        ))}
+        </li>
+
+        <div className="socIndividualBox">
+          Activities
+          <li className="socActivityBox">
```

Comment:
Could you move this activity box styling to the above map( ) with the fetched data. Probably just display the `item.name` for now in the box.